SVProgressHUD/SVProgressHUD.m

```diff
@@ -312,30 +282,28 @@ + (void)resetOffsetFromCenter{
 
 - (instancetype)initWithFrame:(CGRect)frame{
     if((self = [super initWithFrame:frame])){
+        
+        _isInitializing = YES;
+        
         self.userInteractionEnabled = NO;
-        self.backgroundColor = [UIColor clearColor];
+        _backgroundColor = [UIColor clearColor];
```

Comment:
Peter Steinberger explains this in the originally linked article. http://petersteinberger.com/blog/2013/uiappearance-for-custom-views/

The _isInitializing flag prevents the setter from doing anything during init, which allows UIAppearance to work.